en/docs/reference/connectors/db-connector/db-connector-config.md

```diff
@@ -0,0 +1,420 @@
+            <td>Optional</td>
+        </tr>
+        <tr>
+            <td>exhaustedAction</td>
+            <td>Specifies the behavior when the pool is exhausted (e.g., 'FAIL', 'BLOCK', 'GROW').</td>
+            <td>Optional</td>
+        </tr>
+    </table>
+
+    ??? note "Example: MySQL Connection"
+        ```xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <localEntry key="DBCon11" xmlns="http://ws.apache.org/ns/synapse">
+        <db.init>
+            <connectionType>MYSQL</connectionType>
+            <dbUrl>jdbc:mysql://localhost:3306/supermarket</dbUrl>
+            <dbUser>saliya</dbUser>
+            <dbPassword>password</dbPassword>
+            <driverClass>com.mysql.cj.jdbc.Driver</driverClass>
+            <userConsent>true</userConsent>
+            <dbDriver>MySQL</dbDriver>
+            <exhaustedAction>WHEN_EXHAUSTED_BLOCK</exhaustedAction>
+            <name>DBCon11</name>
+        </db.init>
+        </localEntry>
+        ```
+
+    ??? note "Example: PostgreSQL Connection"
+        ```xml
+        <?xml version="1.0" encoding="UTF-8"?>
+        <localEntry key="PostgreSQLCon" xmlns="http://ws.apache.org/ns/synapse">
+        <db.init>
+            <connectionType>POSTGRESQL</connectionType>
+            <dbUrl>jdbc:postgresql://localhost:5432/supermarket</dbUrl>
+            <dbUser>saliya</dbUser>
+            <dbPassword>password</dbPassword>
+            <driverClass>org.postgresql.Driver</driverClass>
+            <userConsent>true</userConsent>
+            <dbDriver>PostgreSQL</dbDriver>
+            <exhaustedAction>WHEN_EXHAUSTED_BLOCK</exhaustedAction>
+            <name>PostgreSQLCon</name>
+        </db.init>
+        </localEntry>
+        ```
+
+## Operations
+
+Operations use the `configKey` attribute to reference a pre-defined connection configuration.
+
+??? note "select"
+    Executes a SQL SELECT query using prepared statements.
+    <table>
+        <tr><th>Parameter Name</th><th>Description</th><th>Required</th></tr>
+        <tr><td>query</td><td>The Raw SQL SELECT statement</td><td>Yes</td></tr>
+        <tr><td>preparedStmt</td><td>The SQL SELECT statement with '?' placeholders for parameters.</td><td>Yes</td></tr>
+        <tr><td>columnNames</td><td>A comma-separated list of column names to be returned in the result set.</td><td>Optional</td></tr>
+        <tr><td>columnTypes</td><td>A comma-separated list of data types for the columns (e.g., VARCHAR, INTEGER, DOUBLE).</td><td>Optional</td></tr>
+        <tr><td>format</td><td>The format of the result set (e.g., JSON, XML).</td><td>Optional</td></tr>
+        <tr><td>table</td><td>The name of the table to be queried.</td><td>Optional</td></tr>
+        <tr><td>columns</td><td>A list of columns to be included in the result set.</td><td>Optional</td></tr>
+        <tr><td>queryType</td><td>The type of query (e.g., online, offline).</td><td>Optional</td></tr>
```

Comment:
Better if we can explain about the online, offline differences in the VSCode Extension somewhere. The user might not have an idea of what is the difference between online and offline.